Build/package-lock.json

```diff
@@ -3742,7 +3742,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
I think I wasn't supposed to commit this file...

Reply:
Yes, please remove the changes to this file.